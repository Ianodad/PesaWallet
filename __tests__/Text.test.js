import React from 'react';
import {shallow} from 'enzyme';
import Text from '../src/components/Text';

describe('<Text/>', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const component = shallow(<Text />);
      expect(component).toMatchSnapshot();
    });

    it('should render with title snapshot', () => {
      const component = shallow(<Text title={'Submit'} />);
      expect(component).toMatchSnapshot();
    });

    it('should render with children', () => {
      const component = shallow(<Text>Header</Text>);
      expect(component).toMatchSnapshot();
    });

    it('should render with style', () => {
      const component = shallow(
        <Text style={{margin: '10px'}} title={'Submit'} />,
      );
      expect(component).toMatchSnapshot();
    });
  });
});
