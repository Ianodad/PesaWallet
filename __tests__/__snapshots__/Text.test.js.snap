// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text/> Rendering should match to snapshot 1`] = `ShallowWrapper {}`;

exports[`<Text/> Rendering should render with children 1`] = `ShallowWrapper {}`;

exports[`<Text/> Rendering should render with style 1`] = `ShallowWrapper {}`;

exports[`<Text/> Rendering should render with title snapshot 1`] = `ShallowWrapper {}`;
